tests/utils/test_dag_processing.py

```diff
@@ -586,13 +586,16 @@ def test_open_maybe_zipped_normal_file_with_zip_in_name(self):
 
     @mock.patch("zipfile.is_zipfile")
     @mock.patch("zipfile.ZipFile")
-    def test_open_maybe_zipped_archive(self, mocked_zip_file, mocked_is_zipfile):
+    @mock.patch("io.TextIOWrapper")
+    def test_open_maybe_zipped_archive(self, mocked_text_io_wrapper, mocked_zip_file, mocked_is_zipfile):
```

Comment:
Can you move this test case to `test/utils/test_file.py`?  During the review, I checked that the file was missing and hence I merged the change that didn't work.

Reply:
Would you like to move `TestCorrectMaybeZipped` to the new `tests/utils/test_file.py` module as well?

Reply:
Yes. Please move it also.